src/bq769x0/bms_bq769x0.c

```diff
@@ -507,7 +508,7 @@ void bms_update_error_flags(BmsConfig *conf, BmsStatus *status)
         error_flags_temp |= 1U << BMS_ERR_DIS_UNDERTEMP;
     }
 
-    status->error_flags = error_flags_temp;
+    status -> error_flags = error_flags_temp;
```

Comment:
Sorry for the additional nits comment, but could you please keep the previous style with `status->error_flags` to maintain consistent coding style everywhere (without space before and after the `->`)? After that we can merge :)

Reply:
Ah. yes, i didnt recognized i discardedy our line and used mine ones. Fixed now. 

PS: 
At least.... git is more complicated to use than fix the real-world problems in sourcecode)